<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratedCommands</class>
 <widget class="QDialog" name="GeneratedCommands">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeneratedCommands</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 56, 70);</string>
  </property>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 20px;
font: 75 13pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="commands">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>651</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 20px;
font: 75 13pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
